system/jlib/jlog.cpp

```diff
@@ -292,28 +293,33 @@ void LogMsgJobInfo::deserialize(MemoryBuffer & in)
 }
 
 static LogMsgJobInfo globalDefaultJobInfo(UnknownJob, UnknownUser);
+static LogMsgTraceInfoId globalDefaultTraceInfoId = UnknownTraceInfoId;
```

Comment:
Not actually used and can be deleted (I will; eventually do the same for the glolbalDefaultJobInfo